def/sk/menu_linux.cson

```diff
@@ -0,0 +1,307 @@
+Menu:
+  "&File":
+    value: "&Súbor"
+    submenu:
+      "New &Window":
+        value: "Nové &okno"
+      "&New File":
+        value: "&Nový súbor"
+      "&Open File…":
+        value: "&Otvorť súbor…"
+      "Open Folder…":
+        value: "Otvoriť zložku…"
+      "Add Project Folder…":
+        value: "Pridať zložku projektu…"
+      "Reopen Project":
+        value: "Znovu otvoriť projekt"
+        submenu:
+          "Clear Project History":
+            value: "Vymazať históriu projektu"
+      "Reopen Last &Item":
+        value: "Znovu otvoriť poslednú &položku"
+      "&Save":
+        value: "Uložiť"
```

Comment:
For menu item containing `&` as hotkey hint, please also add `&` before the letter as your preferred hotkey.
(see no.3 in https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md#translation-guide)

e.g. https://github.com/liuderchi/atom-i18n/blob/83fdc78ee765f2a2cd879b70a0cd634429109d81/def/fr/menu_win32.cson#L5-L6